resources/run_local_service_network.sh

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env bash
+
+set -e
+
+if [ -z "$1" ]; then
+  echo "Error: No count argument provided."
+  echo "Usage: $0 <count>"
+  exit 1
+fi
+count=$1
+
+sudo safenode-manager add --first --local
+sudo safenode-manager start
+
+output=$(sudo safenode-manager status --json)
+
+port=$(echo "$output" | jq -r '.[0].port')
+peer_id=$(echo "$output" | jq -r '.[0].peer_id')
+genesis_multiaddr="/ip4/127.0.0.1/tcp/${port}/p2p/${peer_id}"
+
+sudo safenode-manager add --local --count "$count" --peer "$genesis_multiaddr"
+sudo safenode-manager start
+safenode-manager faucet --peer "$genesis_multiaddr"
```

Comment:
I wonder if this would cause an error with the below check while launching the fuacet
```
            if !local_node_registry.nodes.is_empty() {
                return Err(eyre!("A local network is already running")
                    .suggestion("Use the kill command to destroy the network then try again"));
            }
```
The registry would not be empty as we have added nodes here right?